enclave_manager/setup.py

```diff
@@ -72,16 +72,17 @@
 ]
 
 enclave_module_files = [
-    "avalon_enclave_manager/avalon_enclave.i",
+    "avalon_enclave_manager/avalon_enclave_singleton.i",
     os.path.join(enclave_bridge_wrapper_path, 'swig_utils.cpp'),
     os.path.join(enclave_bridge_wrapper_path, 'work_order_wrap.cpp'),
     os.path.join(enclave_bridge_wrapper_path, 'enclave_info.cpp'),
     os.path.join(enclave_bridge_wrapper_path, 'signup_info.cpp'),
     os.path.join(enclave_bridge_wrapper_path, 'signup_info_singleton.cpp'),
+    os.path.join(enclave_bridge_wrapper_path, 'signup_info_kme.cpp')
```

Comment:
Add a comment that this is temporary and signup_info_kme.cpp will be moved to KME enclave manager later